event/event.go

```diff
@@ -5,12 +5,41 @@ import (
 	"time"
 )
 
+// Category define how to display the Category.
+type Category int32
+
+// These Category enumerations need to be in sync with com_signalfx_metrics_protobuf
+
+const (
+
```

Comment:
So i just tried to integrate this with the proxy, and i think your original idea of using the protobuf constants might be the best plan. Otherwise i have to keep my own map of these in the client too to convert protobuf to these, and then you're converting these to them yourself, as well as everyone having to keep them in sync.  I should have pushed for this when we first talked as keeping the protobuf as a dependency is way easier than keeping maps in sync.